### ISSUE TITLE
Kurseliste automatische Mails an Coach/AL

### ISSUE THREAD

**Issue by ldaschinger** [hitobito/hitobito_pbs#248]
Übers Jahr wollen AL und Coaches den Leitenden Kurse empfehlen welche sie besuchen können mit ihrer aktuellen Ausbildung.
Dafür müssen sie regelmässig die Kursliste auf midata überprüfen ob neue passende Kurse verfügbar sind.
Vorschlag:
- Mit einem opt-in können sich AL/Coach z.bsp jede Woche/Monat eine Liste von neu publizierten Kurse senden lassen plus ein Liste von Leitpersonen/Rover der Organisation welche die Kurse laut stand Ausbildung auf midata besuchen können
- Vereinfachte version wäre die Möglichkeit die Kursliste als csv oder ähnlich herunter zu laden

**Comment by carlobeltrame** [comment]
Spannende Idee. Das könnte man auch in einem externen Tool machen welches die Kursliste via [API](https://github.com/hitobito/hitobito/blob/master/doc/development/05_rest_api.md) abfragt und gegebenenfalls Mails versendet.

**Comment by ldaschinger** [comment]
Ja denke ich, wäre auch schon toll um es dann extern zu verarbeiten.

Gibt es bereits eine Möglichkeit die Kursliste als csv zu exportieren oder ähnlich?

**Comment by carlobeltrame** [comment]
Ja, wie oben verlinkt gibt es eine JSON API (technische Schnittstelle) über welche man Anlässe und Kurse abrufen kann. Ich bin nicht sicher ob da schon alle Filtermöglichkeiten vorhanden sind um diesen Use Case abzudecken, das müsste man ausprobieren.

Falls du das selber ausprobieren möchtest kannst du dich auf der Testumgebung der MiData unter https://pbs.puzzle.ch als Administrator einloggen: hussein_kohlmann@hitobito.example.com / hito42bito
Oder du kannst lokal auf deinem Computer eine MiData laufen lassen, siehe dazu https://github.com/hitobito/development

Falls du mehr Unterstützung brauchst als ich dir hier schriftlich geben kann, oder falls du lieber auf der nicht-technischen Schiene unterwegs bist, kannst du am 24.-26. März nach Baar an den nächsten Hackathon kommen und die Idee ausdiskutieren und allenfalls von anderen begeisterten Pfadis einen Prototypen programmieren lassen.
